include/vctr/Containers/Array.h

```diff
@@ -39,7 +39,8 @@ struct StorageExtent<T, e>
 };
 } // namespace detail
 
-/** The stack-based container type.
+/** @ingroup Core
```

Comment:
Seems that grouping should happen automatically if `GROUP_NESTED_COMPOUNDS` is enabled. Idk though if we always want that. But it's enabled in juce's doxyfile. Maybe we could also use the juce config as a reference.